frontends/verilog/verilog_parser.y

```diff
@@ -218,9 +247,6 @@ static AstNode *checkRange(AstNode *type_node, AstNode *range_node)
 			range_node = makeRange(type_node->range_left, type_node->range_right, false);
 		}
 	}
-	if (range_node && range_node->children.size() != 2) {
-		frontend_verilog_yyerror("wire/reg/logic packed dimension must be of the form: [<expr>:<expr>], [<expr>+:<expr>], or [<expr>-:<expr>]");
-	}
```

Comment:
Did you checked, that now we support packed dimensions in form of the: [<expr>]?

Reply:
I prepared some tests, at least the at seems to be correct, but need to make a SAT on them

Reply:
Edit created tests with SAT works fine